binaries/runtime/src/operator/python.rs

```diff
@@ -47,17 +47,29 @@ pub fn spawn(
             .ok_or_else(|| eyre!("module path has no file stem"))?
             .to_str()
             .ok_or_else(|| eyre!("module file stem is not valid utf8"))?;
-        let module = py
-            .import(module_name)
-            .wrap_err("failed to import Python module")?;
+        let module = py.import(module_name).or_else(|err: pyo3::PyErr| {
+            let trace = err
+                .traceback(py)
+                .expect("PyError should have a traceback")
+                .format()
+                .wrap_err("Traceback could not be formatted")?;
+            Err(eyre!("{trace}"))
```

Comment:
According to https://docs.rs/pyo3/0.16.5/pyo3/types/struct.PyTraceback.html#method.format , the traceback does not contain the actual error. So we should probably do somthing like this:

```suggestion
            Err(eyre!("{err}\n{trace}"))
```

Or

```suggestion
            Err(eyre!("{trace}{err}"))
```

Reply:
Indeed! Good catch! I went for `{trace}\n{err}` as it feels closer to the natural interpreter.